.github/workflows/pr-handling.yaml

```diff
@@ -0,0 +1,38 @@
+name: Assign PR to author and reviewers
+
+# This workflow runs on the pull_request_target event:
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+#
+# So the secrets are accessible even if the PR was opened from an external
+# repository. This is done because the GitHub API key needs to be accessed
+# to modify the PRs.
+#
+# NOTE: Only API calls should be made in the actions defined here. The
+#       committed code should _NOT_ be touched in any case.
+
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
+
+jobs:
+  assign-pr:
+    name: Assign PR to author
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v1.6.0
+
+  ask-review:
+    name: Run pull-review
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run pull-review with docker
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            OWNER: ${{ github.repository_owner }}
+            REPO: ${{ github.event.repository.name }}
+            PULL_REQUEST_NUMBER : ${{ github.event.pull_request.number }}
+        shell: bash
+        run: |
+          PR_URL="https://github.com/${OWNER}/${REPO}/pull/${PULL_REQUEST_NUMBER}"
+          bash scripts/pull-review.sh "${GITHUB_TOKEN}" "${PR_URL}"
```

Comment:
Why not include the content of this script here? Especially since script is just a one-liner starting a docker container.

Reply:
This line is already part of the existing workflow (see the deleted file `.github/workflows/pull-review.yaml` below). It might be a good refactoring to include it directly. But I think this should be done in another PR (to keep the focus of the PR clear).

Reply:
👍  to @paulfantom 's suggestion - given that you will add it as a comment - something that does not modify functionality, I think it is acceptable to include this in this PR.

Reply:
As suggested, I have integrated the script `scripts/pull-review.sh` directly into the workflow.